src/cmd/tools/m3ctl/yaml/examples/replaceNode.yaml

```diff
@@ -0,0 +1,13 @@
+---
+operation: replaceNode
+leavingInstanceIDs:
+- oldnodeid1
```

Comment:
Can you rename this filename to `replace_node.yaml`?  All our file names are lower case under score separated.